python/cuml/test/test_metrics.py

```diff
@@ -160,6 +163,102 @@ def test_rand_index_score(name, nrows):
     assert array_equal(cu_score, cu_score_using_sk)
 
 
+def score_homogeneity(ground_truth, predictions, use_handle):
+    return score_labeling_with_handle(cuml.metrics.homogeneity_score,
+                                      ground_truth,
+                                      predictions,
+                                      use_handle,
+                                      dtype=np.int32)
+
+
+def score_mutual_info(ground_truth, predictions, use_handle):
+    return score_labeling_with_handle(cuml.metrics.mutual_info_score,
+                                      ground_truth,
+                                      predictions,
+                                      use_handle,
+                                      dtype=np.int32)
+
+
+@pytest.mark.parametrize('use_handle', [True, False])
+def test_homogeneity_perfect_labeling(use_handle):
+    # Perfect labelings are homogeneous
+    hom = score_homogeneity([0, 0, 1, 1], [1, 1, 0, 0], use_handle)
+    assert_almost_equal(hom, 1.0, decimal=4)
+    hom = score_homogeneity([0, 0, 1, 1], [0, 0, 1, 1], use_handle)
```

Comment:
I don't think these have moved over to the parameterization style that I mentioned in the other review. I think the target code would look like: 
```
@pytest.mark.parametrize('use_handle', [True, False])
@pytest.mark.parametrize('data', [
  [[0,0,1,1], [1,1,0,0]],
  [[0,0,1,1], [0,0,1,1]] 
 ])
def test_homogeneity_perfect_labeling(use_handle, data):
    # Perfect labelings are homogeneous
    hom = score_homogeneity(data[0], data[1], use_handle)
    assert_almost_equal(hom, 1.0, decimal=4)
```
That separates the data from the test and leaves only one assert